otj-kafka/src/main/java/com/opentable/kafka/logging/LoggingUtils.java

```diff
@@ -218,6 +227,8 @@ private String getHeaderValue(final ConservedHeader header) {
      */
     private String formatHeaders(Headers headers) {
         return Arrays.stream(headers.toArray())
+            // Don't include items that we know go in the otl
```

Comment:
As noted, avoid logging any extraneous headers. So this should consist only of some of the conserved headers.